LPS-82850

topolik opened the pull request:
Hi Carlos,

we need to fix baseline, but I'm not sure what's wrong here.

Try to run: `gradlew :apps:portal-remote:portal-remote-jaxrs-whiteboard:baseline`

Thanks.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

topolik commented:
There is lot of these:
```
:apps:portal-remote:portal-remote-jaxrs-whiteboard:jar
[Split package, multiple jars provide the same package:org/apache/cxf
Use Import/Export Package directive -split-package:=(merge-first|merge-last|error|first) to get rid of this warning
Package found in   [Jar:cxf-core-3.2.5, Jar:classes]
Class path         [Jar:com.liferay.portal.kernel-4.0.0-SNAPSHOT, Jar:javax.json-api-1.0, Jar:org.apache.aries.jax.rs.whiteboard-1.0.0, Jar:org.apache.aries.javax.jax.rs-api-1.0.0, Jar:cxf-core-3.2.5, Jar:org.apache.servicemix.specs.annotation-api-1.3-1.3_1, Jar:org.osgi.service.jaxrs-1.0.0, Jar:org.osgi.util.function-1.1.0, Jar:org.osgi.util.promise-1.1.0, Jar:cxf-rt-bindings-soap-3.2.5, Jar:cxf-rt-bindings-xml-3.2.5, Jar:cxf-rt-databinding-jaxb-3.2.5, Jar:cxf-rt-wsdl-3.2.5, Jar:cxf-tools-validator-3.2.5, Jar:wsdl4j-1.6.3, Jar:classes], Split package, multiple jars provide the same package:org/apache/cxf/annotations
Use Import/Export Package directive -split-package:=(merge-first|merge-last|error|first) to get rid of this warning
Package found in   [Jar:cxf-core-3.2.5, Jar:classes]
Class path         [Jar:com.liferay.portal.kernel-4.0.0-SNAPSHOT, Jar:javax.json-api-1.0, Jar:org.apache.aries.jax.rs.whiteboard-1.0.0, Jar:org.apache.aries.javax.jax.rs-api-1.0.0, Jar:cxf-core-3.2.5, Jar:org.apache.servicemix.specs.annotation-api-1.3-1.3_1, Jar:org.osgi.service.jaxrs-1.0.0, Jar:org.osgi.util.function-1.1.0, Jar:org.osgi.util.promise-1.1.0, Jar:cxf-rt-bindings-soap-3.2.5, Jar:cxf-rt-bindings-xml-3.2.5, Jar:cxf-rt-databinding-jaxb-3.2.5, Jar:cxf-rt-wsdl-3.2.5, Jar:cxf-tools-validator-3.2.5, Jar:wsdl4j-1.6.3, Jar:classes], Split package, multiple jars provide the same package:org/apache/cxf/attachment
Use Import/Export Package directive -split-package:=(merge-first|merge-last|error|first) to get rid of this warning
Package found in   [Jar:cxf-core-3.2.5, Jar:classes]
Class path         [Jar:com.liferay.portal.kernel-4.0.0-SNAPSHOT, Jar:javax.json-api-1.0, Jar:org.apache.aries.jax.rs.whiteboard-1.0.0, Jar:org.apache.aries.javax.jax.rs-api-1.0.0, Jar:cxf-core-3.2.5, Jar:org.apache.servicemix.specs.annotation-api-1.3-1.3_1, Jar:org.osgi.service.jaxrs-1.0.0, Jar:org.osgi.util.function-1.1.0, Jar:org.osgi.util.promise-1.1.0, Jar:cxf-rt-bindings-soap-3.2.5, Jar:cxf-rt-bindings-xml-3.2.5, Jar:cxf-rt-databinding-jaxb-3.2.5, Jar:cxf-rt-wsdl-3.2.5, Jar:cxf-tools-validator-3.2.5, Jar:wsdl4j-1.6.3, Jar:classes], Split package, multiple jars provide the same package:org/apache/cxf/binding
Use Import/Export Package directive -split-package:=(merge-first|merge-last|error|first) to get rid of this warning
Package found in   [Jar:cxf-core-3.2.5, Jar:classes]
```

And then those:
```
:apps:portal-remote:portal-remote-jaxrs-whiteboard:baseline
  PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
= ================================================== ========== ========== ========== ========== ==========
  org.apache.cxf                                     UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
        -   version    3.2.4
        +   version    3.2.5
  org.apache.cxf.annotations                         UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
        -   version    3.2.4
        +   version    3.2.5
  org.apache.cxf.attachment                          UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
        -   version    3.2.4
        +   version    3.2.5
  org.apache.cxf.binding                             UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
        -   version    3.2.4
        +   version    3.2.5
  org.apache.cxf.binding.soap                        UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
        -   version    3.2.4
        +   version    3.2.5
  org.apache.cxf.binding.soap.interceptor            UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
        -   version    3.2.4
        +   version    3.2.5
  org.apache.cxf.binding.soap.jms.interceptor        UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
        -   version    3.2.4
        +   version    3.2.5
  org.apache.cxf.binding.soap.model                  UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
        -   version    3.2.4
        +   version    3.2.5
  org.apache.cxf.binding.soap.saaj                   UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
        -   version    3.2.4
        +   version    3.2.5
  org.apache.cxf.binding.soap.wsdl.extensions        UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
        -   version    3.2.4
        +   version    3.2.5
  org.apache.cxf.binding.soap.wsdl11                 UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
        -   version    3.2.4
        +   version    3.2.5
  org.apache.cxf.binding.xml                         UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
        -   version    3.2.4
        +   version    3.2.5
  org.apache.cxf.binding.xml.interceptor             UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
        -   version    3.2.4
        +   version    3.2.5
  org.apache.cxf.binding.xml.wsdl11                  UNCHANGED  3.2.5      3.2.4      3.2.4      EXCESSIVE VERSION INCREASE
```
